.github/workflows/r_valgrind.yml

```
@@ -7,7 +7,7 @@ on:
 jobs:
   test-r-valgrind:
     name: r-package (ubuntu-latest, R-devel, valgrind)
-    timeout-minutes: 120
+    timeout-minutes: 180
```

```suggestion
    timeout-minutes: 120
```

Apologies, I misunderstood how this check works. This config is sourced from `master` when the relevant CI job runs, so changing it here won't work.

https://github.com/microsoft/LightGBM/pull/4397#issuecomment-866790489

Once #4404 is merged, I'll test this again.